### ISSUE TITLE
Error thrown when removing selections

### ISSUE THREAD

**Issue by blazingdrummer** [zachrip/valpal#5]
Pops up fairly often, can't seem to replicate it consistently tho
![image](https://github.com/zachrip/valpal/assets/3999293/1b723ee6-7bc9-443e-a1a4-a370a60f7a49)

**Comment by blazingdrummer** [comment]
refreshing the page gets things working again quickly tho

**Comment by zachrip** [comment]
@blazingdrummer what do you mean by removing selections? Just removing any skin selection?

**Comment by blazingdrummer** [comment]
Yep exactly. When you click on a previously selected skin in order to remove it.
